Makefile

```diff
@@ -127,6 +127,16 @@ virtualenv_ansible:
 		fi; \
 	fi
 
+virtualenv_ansible_py3:
```

Comment:
so this won't be in the default image as of now?

Reply:
the default dev container image, yep

Reply:
Wouldn't you then need to `make virtualenv_ansible_py3` in the Dockerfile?

Reply:
ooh, you added it to `requirements_dev`...

Reply:
but if the Docker file remains unchanged, will it actually rebuild the image, or use the cached version, because each step is cached

Reply:
Well, given that zuul tests are passing using this code, I'd expect an image is being built/updated (because my new tests won't pass if `/venv/ansible3` doesn't exist.